components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityUtil.java

```diff
@@ -225,7 +224,7 @@ public static String getPPIDDisplayValue(String value) throws Exception {
         if (log.isDebugEnabled()) {
             log.debug("Generating display value of PPID : " + value);
         }
-        byte[] rawPpid = Base64.decode(value);
```

Comment:
We can use jdk decoder as well?

Reply:
Yes. Fixed with https://github.com/wso2/carbon-identity-framework/pull/2921/commits/7b2708081125a410e0d660cf4f065fd3131049fd.